pyproject.toml

```diff
@@ -43,13 +43,23 @@ flake8 = ">=3.9.2"
 hvac = ">=0.11.0, <1.1.0"  # (always keep in sync w/ optional dependency)
 invoke = "*"
 moto = "~2.2.11"
-mkdocs = "*"
 pydocstyle = "*"
 pylint = "*"
 pylint-django = "*"
 python-tss-sdk = "~1.2.0"  # (always keep in sync w/ optional dependency)
 requests-mock = "~1.9.3"
 yamllint = "*"
+Markdown = "*"
+toml = "*"
+# Rendering docs to HTML
+mkdocs = "1.3.1"
+# Material for MkDocs theme
+mkdocs-material = "8.4.2"
+# Render custom markdown for version added/changed/remove notes
+mkdocs-version-annotations = "1.0.0"
+# Automatic documentation from sources, for MkDocs
+mkdocstrings = "0.19"
+mkdocstrings-python = "0.7.1"
```

Comment:
Normally we've tried to keep dependencies in alphabetical order in `pyproject.toml` for ease of maintenance.

Reply:
https://github.com/networktocode-llc/cookiecutter-ntc/blob/nautobot-plugin-v1.0/nautobot-plugin/%7B%7B%20cookiecutter.project_slug%20%7D%7D/pyproject.toml#L30C1-L30C1

Reply:
Will sort it in the templates, keeping it here for now.